spack/packages/dla-future/package.py

```diff
@@ -16,23 +15,41 @@ class DlaFuture(CMakePackage):
 
     version('develop', branch='master')
 
-    variant('gpu', default=False,
+    variant('cuda', default=False,
             description='Use the GPU/cuBLAS back end.')
-
-    # Until mpich is default comment this out
+    variant('test', default=False,
+            description='Run unit tests.')
+    variant('doc', default=False,
+            description='Build documentation.')
+    
     #depends_on('mpi@3:')
-    depends_on('mpich')
-    depends_on('mkl')
+    depends_on('mpi')
+    depends_on('blas')
+    depends_on('lapack')
```

Comment:
I'm a bit concerned on this, I think we should depend just in blaspp/lapackpp and use the one used by them.

Reply:
Depending on `blaspp` and `lapackpp` for now is probably sufficient. We can add explicit blas and lapack dependence if we start depending directly on a BLAS/LAPACK library directly. However, I'm OK with leaving them too if the package still works, the concretizer should be able to provide a consistent dependency tree regardless.

Reply:
I have tried and it works perfectly without these dependencies, as you both suggested. I will remove them. Thanks.